Apps/Patient/src/Delegates/IClientRegistriesDelegate.cs

```diff
@@ -0,0 +1,37 @@
+//-------------------------------------------------------------------------
+// Copyright © 2019 Province of British Columbia
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//-------------------------------------------------------------------------
+namespace HealthGateway.Patient.Delegates
+{
+    using System.Threading.Tasks;
+    using HealthGateway.Common.Constants;
+    using HealthGateway.Common.Data.ViewModels;
+    using HealthGateway.Common.Models;
+
+    /// <summary>
+    /// The Patient data service.
+    /// </summary>
+    public interface IClientRegistriesDelegate
+    {
+        /// <summary>
+        /// Gets the patient record.
+        /// </summary>
+        /// <param name="type">The oid type value.</param>
+        /// <param name="value">The associated oid type's value to retrieve the patient demographics information.</param>
+        /// <param name="disableIdValidation">Disables the validation on HDID/PHN when true.</param>
+        /// <returns>The patient model wrapped in an api result object.</returns>
+        Task<ApiResult<PatientModel>> GetDemographicsAsync(OidType type, string value, bool disableIdValidation = false);
```

Comment:
Suggest renaming parameter `value` to `identifier`.

Reply:
ok